views/event.ejs

```diff
@@ -23,53 +23,91 @@
+          <img src="/<%= event.picture %>" class="img-fluid" />
+        </div>
+        <div class="col-md-7 order-2 order-md-1">
+          <p>
+            <%= moment(event.startDate).locale('de').format('L') %>,
+            <%= moment(event.startDate).locale('de').format('LT') %>
+            -
+            <%= moment(event.endDate).locale('de').format('LT') %>
+          </p>
+          <p>
+            <% if (locals.event) { %>
+              <%= event.description %>
+            <% } %>            
+          </p>
+          <p>Kategorie: 
+            <% if (locals.event.participants) {  %>
+              <% for(var i=0; i<locals.event.categories.length; i++) { %>
+                <span class="event-category"><%= locals.event.categories[i] %></span>
               <% } %>
+            
+            <% } %>            
+          <br>
+          Schwierigkeit: <%= locals.event.difficulty %>
+          <br>
+          Sprache: <%= locals.event.language %>
+
+          </p>
+          <p> 
+            <% if (locals.event.participants) {  %>
+              <% for(var i=0; i<locals.event.tags.length; i++) { %>
+                <span class="event-tag">#<%= locals.event.tags[i] %></span>
               <% } %>
-            </ul>
-          </div>
-          <div class="mt-5 text-left">
-            <h3>Teilnehmer</h3>
-            <% if (locals.event.participants) { %>
-            <i class="fa fa-id-card"></i> <%= event.participants.length %>
-            <% } %>
-          </div>
+            
+            <% } %>            
+          </p>    
         </div>
-      </div>
-      <div class="row">
-        <div class="col-lg-12">
-          <h3>Beschreibung
-          </h3>
-          <% if (locals.event) { %>
-          <%= event.description %>
-          <% } %>
+      </div>      
+          <div class="row">
+            <div class="col-12">
+              <p>
+            <h6>Teilnehmer</h6>
+            <% if (locals.event.participants) {  %>
+              <% for(var i=0; i<locals.event.participants.length; i++) { %>
+                <!-- In future versions this should be the circular profile image -->
+                <!-- <img class="rounded-circle event-participant-image" src=<%= locals.event.participants[i].profilePic %> /> -->
+                <span class="event-image-circle"><%= locals.event.participants[i].name[0] %></span>
+              <% } %>
+            
+            <% } else { %>
+              Noch keine Teilnehmer. Gleich anmelden! <%= event.trainer.name %> freut sich.
+              <% } %>
+            </p>
+          <p>
+            <% if (locals.event.groups.length > 0) {  %>
+              <h6>Teil folgender Gruppen</h6>
+              <% for(var i=0; i<locals.event.groups.length; i++) { %>
+                <!-- In future versions this should be the circular profile image -->
+                <!-- <img class="rounded-circle event-participant-image" src=<%= locals.event.participants[i].profilePic %> /> -->
```

Comment:
hier fehlt das "/" noch

Reply:
Das ist jetzt in den Testdaten enthalten